main.go

```diff
@@ -72,17 +75,23 @@ func main() {
 		log.Fatal().Err(err).Msg("could not initialize mapper")
 	}
 
-	controller, err := rest.NewController(core)
+	restController, err := rest.NewController(core)
```

Comment:
I would still like to avoid the long name. You could use `rctrl` and `gctrl`. Or `restc` and `grpcc`. Thoughts?